pkg/admissioncontroller/webhook/auth/seed/graph/eventhandler_managedseed.go

```diff
@@ -101,6 +102,12 @@ func (g *graph) handleManagedSeedCreateOrUpdate(ctx context.Context, managedSeed
 		}
 	}
 
+	if metav1.HasAnnotation(managedSeed.ObjectMeta, seedmanagementv1alpha1constants.AnnotationSeedSecretName) &&
+		metav1.HasAnnotation(managedSeed.ObjectMeta, seedmanagementv1alpha1constants.AnnotationSeedSecretNamespace) {
+		secretVertex := g.getOrCreateVertex(VertexTypeSecret, managedSeed.GetAnnotations()[seedmanagementv1alpha1constants.AnnotationSeedSecretNamespace], managedSeed.GetAnnotations()[seedmanagementv1alpha1constants.AnnotationSeedSecretName])
+		g.addEdge(secretVertex, managedSeedVertex)
+	}
```

Comment:
Hmm, in https://github.com/gardener/gardener/blob/5821b96e0852c1b7c34a39112f1657c17a1ff5c9/pkg/admissioncontroller/webhook/auth/seed/graph/eventhandler_managedseed.go#L99-L102 I think we do the same. Isn't it the same?

Reply:
Yes, but I didn't want to change the existing behaviour, the edge is not duplicated, as it's visible in the tests, https://github.com/gardener/gardener/blob/5821b96e0852c1b7c34a39112f1657c17a1ff5c9/pkg/admissioncontroller/webhook/auth/seed/graph/graph_test.go#L988-L1012

Reply:
I now see the reason. So it is because https://github.com/gardener/gardener/blob/5821b96e0852c1b7c34a39112f1657c17a1ff5c9/pkg/admissioncontroller/webhook/auth/seed/graph/eventhandler_managedseed.go#L99-L102 won't be executed when secretRef is nil and https://github.com/gardener/gardener/blob/5821b96e0852c1b7c34a39112f1657c17a1ff5c9/pkg/admissioncontroller/webhook/auth/seed/graph/eventhandler_managedseed.go#L105-L109 will be executed because the annotations will be still there after the secretRef is nil?